api/reviews_api.py

```diff
@@ -163,6 +163,52 @@ def validate_assignees(self, assignees, fe, gate):
         self.abort(400, 'Assignee is not a reviewer')
 
 
+class PendingGatesAPI(basehandlers.APIHandler):
+
+  def get_stage_ids_of_gates_pending_my_approval(self) -> list[int]:
```

Comment:
This return type might not be correct if returning the result of `fetch_async()`.

Reply:
I updated several return types to be `list[int] | Future`.